tests/test_globalization.py

```diff
@@ -168,61 +164,60 @@ def run_charset_validation(self, charset):
             shutdown(client)
 
 
-class GlobalizationMetadataTests(GlobalizationTests, unittest.TestCase):
-    @classmethod
-    def setUpClass(cls):
-        cls.test_db = create_test_db()
-
-    @classmethod
-    def tearDownClass(cls):
-        clean_up_test_db(cls.test_db)
+class TestGlobalizationMetadata(GlobalizationTests):
```

Comment:
Remind me: Does pytest detect tests by convention? E.g. it finds anything starting with Test and treats as test class?

Reply:
Exactly as you described--`test` must prefix class/function names.